### PR TITLE
🔨 chore: add command to prune node_modules

### PR DESCRIPTION
**Pull request by samrichca**
- add a command that lists then removes node_modules, and reinstalls them.
- update readme

### CODE REVIEW

**Comment by alyssadsouza** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -120,6 +120,20 @@ In addition to downloading VS Code, please ensure that you have the following VS
 
 Please note that we are currently specifying versions of `@typescript-eslint/parser` and `eslint-plugin-import` in the `overrides` section of the root [`package.json` file](./package.json). This was done to avoid a conflict between the version used by the `eslint-config-next` plugin and our custom ESLint configuration.
 
+### How to reset node_modules
+
+To execute `rm -rf node_modules` on all repos in this monorepo:
+
+```sh
+npm run reset-node-modules
```
Comment (mistryrn) [inline README.md:128]: Is this `npm` instead of `pnpm` on purpose?

**Review by alyssadsouza — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (samrichca) [reply README.md]: good eye, updated

**Comment by ciaranschutte** [comment]
maybe add a note that this only works on Unix systems?
https://www.npmjs.com/package/rimraf will get you cross platform recursive deletion
likely there's a cross platform `find` package too.
but at that point it's more of a local config thing

**Comment by anncatton** [comment]
Closing, after discussion with the team these changes are not needed

**Pull request closed without merging**